src/cases/containers/role-and-access/role-access.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component, OnInit } from '@angular/core';
+import { RoleExclusion } from '../../../cases/models/role-exclusions/role-exclusion.model';
+import { RoleExclusionsService } from '../../../cases/services';
+
+@Component({
+    selector: 'exui-role-access',
+    templateUrl: './role-access.component.html'
+  })
+
+export class RoleAccessComponent implements OnInit {
+  constructor(private readonly roleExclusionsService: RoleExclusionsService) { }
+  public exclusions: RoleExclusion[];
+
+  public ngOnInit() {
```

Comment:
There is no need of manually subscribe to `getCurrentUserRoleExclusions()`.  Can we take advantage of the `async` pipe

Reply:
Agree